.travis.yml

```diff
@@ -61,14 +61,10 @@ before_install:
 install:
   - npm ci
   - composer install
-  - WCPAY_DIR="/home/travis/build/Automattic/woocommerce-payments/" bash ./bin/install-wp-tests.sh woocommerce_test root '' localhost $WP_VERSION false
+  - WCPAY_DIR="/home/travis/build/Automattic/woocommerce-payments/" bash ./bin/install-wp-tests.sh woocommerce_test root '' localhost $WP_VERSION $WC_VERSION false
 
 script:
   - bash bin/phpunit.sh
   - bash bin/phpcs.sh
   - npm run lint
   - npm test
-
-branches:
-  only:
-    - master
```

Comment:
It annoyed me that Ci wasn't running in this PR because the target branch isn't `master`. To prevent a single push to trigger 2 builds, I disabled `Build pushed branches` in Travis:
![Screenshot 2020-08-02 at 08 27 22](https://user-images.githubusercontent.com/1715800/89117984-1aeccc80-d49a-11ea-954f-9ae02d94b987.png)

The only downside is that now (I think) Travis won't run on commits pushed/merged to `master`. Are we OK with that trade-off?

Reply:
Good idea! Changed it in https://github.com/Automattic/woocommerce-payments/pull/813/commits/bd730bb35d3f361791cf2f803f36f023e4676488 (and enabled `Build pushed branches` on Travis again).